include/wx/qt/dataobj.h

```diff
@@ -12,13 +12,20 @@
 // wxDataObject is the same as wxDataObjectBase under wxQT
 // ----------------------------------------------------------------------------
 
+class QMimeData;
+
 class WXDLLIMPEXP_CORE wxDataObject : public wxDataObjectBase
 {
 public:
     wxDataObject();
     virtual ~wxDataObject();
 
     virtual bool IsSupportedFormat( const wxDataFormat& format, Direction dir = Get ) const;
+	virtual void AddDataTo(QMimeData &mimeData) const;
+	virtual bool SetDataFrom(const QMimeData &mimeData);
+
+private:
+	virtual void DoSetDataFrom(const QMimeData &mimeData, const wxDataFormat &format);
```

Comment:
I wouldn't call this `DoSetDataFrom` (or `QtDoSetDataFrom`) as `DoFoo()` conventionally means that this function is the real implementation of the public `Foo()`. Here this is not the case because `SetDataFrom()` may call `DoSetDataFrom()` or not. So I'd just call this `SetDataSingleFormat()` or something like this.

I also think that a comment explaining the relationships between these functions, and why do we need to override them in text/bitmap daya objects, would be really, really helpful.